src/qt/res/css/trad.css

```
@@ -1,29 +1,68 @@
-/* MODAL OVERLAY */
+/**
+Copyright (c) 2020 The Dash Core developers
+Distributed under the MIT/X11 software license, see the accompanying
+file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-QWidget#bgWidget { /* The 'frame' overlaying the overview-page */
-background: rgba(0,0,0,220);
+---------------------------------------------------------------------
+
+This file contains all changes for the dash theme "Traditional".
+
+NOTE: This file gets not appended to the general.css. The Traditional
+theme is a standalone theme which just fixes some bugs in the OS default
+depiction of the Qt elements.
+
+Loaded in GUIUtil::loadStyleSheet() in guitil.cpp.
+**/
+
+/* do not modify! section updated by update-css-files.py
+<colors>
+
+# Used colors in trad.css for commit 2c577f004
+
+#fff
+#fafafacc
+
+</colors>
+*/
+
+/******************************************************
+ModalOverlay
+******************************************************/
+
+QWidget#bgWidget { /* The frame overlaying the overview-page */
+    padding-left: 10px;
+    padding-right: 10px;
+    background-color: #fafafacc;
```

This results in some weird yellow-ish background. I guess you messed up with the alpha channel here, probably want smth like
```suggestion
    background-color: #ccfafafa;
```
instead (a slightly transparent white-ish one).

---

Oh right, good one, had that issue in the other files too.. seems like it forgot to fix it in the trad.css 👍